### PR TITLE
migration and seed data for locations and trips

### PR DESCRIPTION
**Pull request by jeelnaik**
pending review

### CODE REVIEW

File: app/assets/stylesheets/locations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Locations controller here.
```
Comment (winescout) [inline app/assets/stylesheets/locations.scss]: We can delete this file.  We won’t need it

File: app/assets/stylesheets/trips.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Trips controller here.
```
Comment (winescout) [inline app/assets/stylesheets/trips.scss]: Don’t need this file either

File: app/helpers/locations_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module LocationsHelper
```
Comment (winescout) [inline app/helpers/locations_helper.rb]: Don’t need this file

File: app/helpers/trips_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module TripsHelper
```
Comment (winescout) [inline app/helpers/trips_helper.rb]: Don’t need this file

File: config/routes.rb [hunk 5]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :locations
```
Comment (winescout) [inline config/routes.rb]: Both of these resources lines need to go below the “get ...” on line 5, but before the “root to ...” on 6

File: app/assets/stylesheets/locations.scss (hunk 1 above)
Comment (jeelnaik) [reply app/assets/stylesheets/locations.scss]: file deleted

File: app/helpers/locations_helper.rb (hunk 3 above)
Comment (jeelnaik) [reply app/helpers/locations_helper.rb]: file deleted

File: config/routes.rb (hunk 5 above)
Comment (jeelnaik) [reply config/routes.rb]: made the changes

File: app/helpers/trips_helper.rb (hunk 4 above)
Comment (jeelnaik) [reply app/helpers/trips_helper.rb]: file deleted

File: app/assets/stylesheets/trips.scss (hunk 2 above)
Comment (jeelnaik) [reply app/assets/stylesheets/trips.scss]: file deleted

**Pull request merged**